README.md

```
@@ -49,6 +49,21 @@ Use bbalet / bbalet for login / password.
+
+This repository includes a Docker environment for running the application with a PHP/Laravel backend, a Node/Vite frontend and a MySQL database.
+
+1. Copy `backend/.env.example` to `backend/.env` and adjust the variables.
+2. Copy `frontend/.env.example` to `frontend/.env` and update the API endpoint if needed.
+3. Build and start the services:
+
+   ```bash
+   docker compose up --build
+   ```
+
+The backend will be available on http://localhost:8000 and the frontend on http://localhost:3000.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)